bakery/src/cli/execute.js

```diff
@@ -309,9 +309,10 @@ const tasks = {
       const buildExec = path.resolve(BAKERY_PATH, 'build')
 
       const imageDetails = imageDetailsFromArgs(argv)
+      const nebGetFlags = argv.requestLimit == null ? '' : `--request-limit ${argv.requestLimit}`
       const taskArgs = imageDetails == null
         ? []
-        : [`--taskargs=${JSON.stringify(imageDetails)}`]
+        : [`--taskargs=${JSON.stringify({ ...imageDetails, ...{ nebGetFlags: nebGetFlags } })}`]
```

Comment:
```suggestion
        : [`--taskargs=${JSON.stringify({ ...imageDetails, nebGetFlags: nebGetFlags })}`]
```
No need for spread operator here.